### PR TITLE
R doc 1593 scripts

### PR DESCRIPTION
**Pull request by morhilai**
Mentioned the scripts: `setup-as-service.ps1`, `uninstall-service.ps1`, and `install-daemon.sh`.
In the 4.2 docs: `Getting Started` > `Installation`> `Running as a Service` and `Deployment Considerations`.

### CODE REVIEW

**Comment by arekpalinski** [comment]
Please check travis CI build error:

```
Unhandled Exception: System.InvalidOperationException: DiscussionId is missing in C:\Users\travis\build\ravendb\docs\Documentation\4.2\Raven.Documentation.Pages\start\installation\.docs.json for item: deployment-considerations. Please generate the documentation again and make sure to commit the .docs.json file changes.
```

You can ask @topalach if needed

**Comment by morhilai** [comment]
Fixed

File: Documentation/4.2/Raven.Documentation.Pages/start/installation/deployment-considerations.markdown [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+## Storage Considerations
+
+The requirements of the storage can be found in the article describing [Voron](../../server/storage/storage-engine#requirements), the storage engine used by RavenDB.
+
+{NOTE: SMB / CIFS is not supported for production scenarios}
+
+[SMB / CIFS](https://en.wikipedia.org/wiki/Server_Message_Block) is a protocol that can be used to access remote files over then network, between machines running different operating systems.
+It is widely used in two scenarios:
+
+-  Azure File Storage service aka [CloudStor:Azure](https://azure.microsoft.com/en-us/services/storage/files/)
```
Comment (ayende) [inline Documentation/4.2/Raven.Documentation.Pages/start/installation/deployment-considerations.markdown]: Should this be CloudStor*e*?

Comment (maximburyak) [reply Documentation/4.2/Raven.Documentation.Pages/start/installation/deployment-considerations.markdown]: CloudStor is correct: https://docs.docker.com/docker-for-azure/persistent-data-volumes/

**Pull request merged**